src/main/java/com/crud/tasks/domain/CreatedTrelloCard.java

```diff
@@ -0,0 +1,26 @@
+package com.crud.tasks.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@Getter
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CreatedTrelloCard {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("shortUrl")
+    private String shortUrl;
+
+    //@JsonProperty("badges")
```

Comment:
czemu zakomentowane?

Reply:
To było potrzebne do jednego z zadań. Myślałem, że jeszcze będzie używane, ale dalej w kursie nie ma już tego pola więc kasuję.